guides/v2.10.0/tutorial/simple-component.md

```diff
@@ -159,8 +159,25 @@ export default Ember.Component.extend({
 });
 ```
 
-To allow the user to widen the image, we will need to add an action that toggles the value of `isWide`.
-Let's call this action `toggleImageSize`
+To allow the user to widen the image, we will need to add an action that toggles
+the value of `isWide`. Let's create the `toggleImageSize` function to toggle the
```

Comment:
I think it would be really cool to already name the `toggleImageSize` action as such here to make it sound more distinct from a regular function via e.g.

```suggestion
the value of `isWide`. Let's create the `toggleImageSize` action to toggle the
```

or 

```suggestion
the value of `isWide`. Let's create the `toggleImageSize` function and assign it to the component's action hash to toggle the
```

What do you think?